### PR TITLE
Ensuring server stops after each test case

### LINKED ISSUE #27874: Test Failure: com.ibm.ws.logging.fat.TestHideMessages.testEmptyHideMessageAttribute
RTC 295597

```
testEmptyHideMessageAttribute:java.lang.Exception: 2023-04-17-17:14:42:127 Errors/warnings were found in server com.ibm.ws.logging.hidemessage logs:
<br>[4/17/23, 17:14:41:418 PDT] 00000034 com.ibm.ws.runtime.update.internal.RuntimeUpdateManagerImpl  W CWWKE1102W: The quiesce operation did not complete. The server will now stop.
<br>[4/17/23, 17:14:41:418 PDT] 00000034 com.ibm.ws.runtime.update.internal.RuntimeUpdateManagerImpl  W CWWKE1105W: 2 runtime updates did not complete during the quiesce period.
    at componenttest.topology.impl.LibertyServer.checkLogsForErrorsAndWarnings(LibertyServer.java:3108)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:2967)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:2850)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:2759)
    at componenttest.topology.impl.LibertyServer.stopServer(LibertyServer.java:2734)
    at com.ibm.ws.logging.fat.TestHideMessages.cleanupTest(TestHideMessages.java:171)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:204)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:364)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:178)
```

The frequency of this problem is about once per month.

### PR DESCRIPTION
**Pull request by tonyreigns**
Fixes: #27874

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

File: dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/TestHideMessages.java [hunk 1]
```diff
@@ -56,6 +56,14 @@ public void setupTest() throws Exception {
         msgServer.startServer();
     }
 
+    @After
+    public void cleanUp() throws Exception {
+        if (msgServer != null && msgServer.isStarted()) {
+            msgServer.stopServer("CWMCG0007E", "CWMCG0014E", "CWMCG0015E",
+                                 "CWMCG5003E", "CWPMI2006W", "CWMMC0013E", "CWWKG0033W");
```
Comment (pgunapal) [inline dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/TestHideMessages.java]: Are these Warning/Error messages related to the test cases? It seems like some are from MP Metrics...

**Review by pgunapal — APPROVED** [review]
LGTM

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
Cancelled and restarted builds to include updated changes.

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
Ci-Orchestrator build's been failing due to a known issue with "impossible buckets". RTC builds have been green.

#libby

**Pull request merged**